src/test/java/com/commercetools/sync/products/helpers/VariantReferenceResolverTest.java

```diff
@@ -267,8 +286,9 @@ void resolveReferences_WithMixedReferences_ShouldResolveReferenceAttributes() {
         // assertions
         final List<AttributeDraft> resolvedBuilderAttributes = resolvedBuilder.getAttributes();
 
-        assertThat(resolvedBuilderAttributes).hasSize(3);
+        assertThat(resolvedBuilderAttributes).hasSize(5);
         assertThat(resolvedBuilderAttributes).contains(textAttribute);
+        assertThat(resolvedBuilderAttributes).isNotNull();
```

Comment:
yes, that is true. I actually did it intentionally, since the line after IDEA was giving me a warning that `resolvedBuilderAttributes.get(0);` could product an NPE.

But I agree, I shouldn't be adding code just to remove an IDEA warning: https://github.com/commercetools/commercetools-sync-java/pull/444/commits/438c440d25de9f090f3047933b9af139991f917b